src/item/BoatType.php

```diff
@@ -69,4 +72,8 @@ public function getWoodType() : WoodType{ return $this->woodType; }
 	public function getDisplayName() : string{
 		return $this->woodType->getDisplayName();
```

Comment:
Wouldn't it be better to combine suffix and wood type directly in this function rather than elsewhere?